site/turbo.json

```diff
@@ -5,6 +5,12 @@
       "outputs": ["dist/**", "sidebar/**"],
       "dependsOn": ["^build", "^docs:gen"],
       "cache": false
+    },
+    "dev": {
+      "dependsOn": ["^build"],
+      "persistent": true,
+      "cache": false,
+      "interruptible": true
```

Comment:
this doesn't get hot reloaded when packages change, so this one has to be restarted by turbo